src/client/datascience/jupyter/jupyterExecution.ts

```diff
@@ -202,23 +199,7 @@ export class JupyterExecutionBase implements IJupyterExecution {
                         throw err;
                     }
                 }
-            }
-
-            // Note: This is unlikely, so far only 1 telemetry captured for this.
-            // If we're here, then starting jupyter timeout.
-            // Kill any existing connections.
-            connection?.dispose();
```

Comment:
This has never been hit , we have just 1 telemetry event for this in the past 6 months.
What happens is we just time out and the error is bubbled up.